### DISCUSSION TITLE
有IOS版本么，能导入录音么

### DISCUSSION THREAD

**Discussion by wuziqun** [ZuodaoTech/everyone-can-use-english#687] [Ideas]
都是用手机，有IOS版本计划么，手机上录的录音，可以导入么？

**Comment by an-lee** [comment]
目前暂时只有电脑版本。
录音导入是什么意思？Enjoy 支持自行添加音视频作为练习材料。

**Reply by wuziqun** [reply]
我知道添加音视频作为练习材料是可以的。但无法添加音频录音材料，只能电脑端录音，事实上，我经常在手机端练习并录音，想把这些录音材料，导入PC端来分析。目前来看是不支持的。如果有此方面计划，我可以参与编程。

**Reply by an-lee** [reply]
v0.2.12 版支持上传录音文件进行发音评估了，如果这是你说的分析。

**Reply by wuziqun** [reply]
我未能在录音管理中找到导入功能（V0.2.12版本）。我只能导入音频（AI生成）作为练习材料，我的跟读录音只能在PC端，在线录音。无法将我在手机端录好的录音资料导入。
![enjoy](https://github.com/xiaolai/everyone-can-use-english/assets/164479577/8382e5f0-fa83-4df3-b846-8eacf8ee09b7)
